manager/eris-mint/evm/vm.go

```diff
@@ -565,21 +566,28 @@ func (vm *VM) call(caller, callee *Account, code, input []byte, value int64, gas
 			}
 			acc := vm.appState.GetAccount(addr)
 			if acc == nil {
-				return nil, firstErr(err, ErrUnknownAddress)
+				if _, ok := registeredNativeContracts[addr]; !ok {
+					return nil, firstErr(err, ErrUnknownAddress)
+				}
+				stack.Push64(int64(One256))
+				dbg.Printf(" => Hit native contract\n\n")
+			} else {
+				code := acc.Code
+				l := int64(len(code))
+				stack.Push64(l)
```

Comment:
again, I think this should be `stack.Push(One256)` it saves doing the conversion to use the constant, and it is used elsewhere